Lib/defconQt/glyphView.py

```diff
@@ -187,27 +292,38 @@ def __init__(self, glyph, parent=None):
         glyphMenu.addAction("&Jump", self.changeGlyph, "J")
         menuBar.addMenu(glyphMenu)
 
-        toolBar = QToolBar(self)
+
+        self._displaySettings = DisplayStyleSettings("&Display", self, self._displayChanged)
+        menuBar.addMenu(self._displaySettings.menuWidget)
+
+        self._toolBar = toolBar = QToolBar(self)
         toolBar.setMovable(False)
         toolBar.setContentsMargins(2, 0, 2, 0)
-        selectionToolButton = toolBar.addAction("Selection", self.view.setSceneSelection)
+        selectionToolButton = toolBar.addAction("Selection", self._redirect('view', 'setSceneSelection'))
         selectionToolButton.setCheckable(True)
         selectionToolButton.setChecked(True)
         selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
-        penToolButton = toolBar.addAction("Pen", self.view.setSceneDrawing)
+        penToolButton = toolBar.addAction("Pen", self._redirect('view', 'setSceneDrawing'))
         penToolButton.setCheckable(True)
-        penToolButton.setIcon(QIcon(":/resources/curve.svg"))
-        rulerToolButton = toolBar.addAction("Ruler", self.view.setSceneRuler)
+        selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
+        rulerToolButton = toolBar.addAction("Ruler", self._redirect('view', 'setSceneRuler'))
```

Comment:
Make the toolbar (and all UI elements controlled by the view) be created in the View and returned with a getter function.  
e.g. see what I did with `view._currentLayerBox` in the View, then the MainGfxWindows calls view.currentLayerBox() to get the widget and put it in the layout.

This should allow removing the _redirect function.